forte/data/data_pack.py

```diff
@@ -787,13 +787,13 @@ def get_data(
                         },
                     base_ontology.Token: ["pos", "sense""],
                     base_ontology.EntityMention: {
-                        "unit": "Token",
                     },
                 }
                 pack.get_data(base_ontology.Sentence, requests)
 
         Args:
-            context_type (str): The granularity of the data context, which
+            context_type (Union[str, Type[Annotation], Type[AudioAnnotation]]):
```

Comment:
No, just adding the types here, is not enough. There are far more behavior changes, here are a couple examples:
1. line 797 here is wrong.
2. In this [line](https://github.com/asyml/forte/pull/603/files#diff-867f9304e792cf6a933c0411a8654c1497b3b3d61fe0fec246d2159ca9884868R1081), the return key changes, which is a piece of very important information to show in the docstring. 

There are a lot more, you should consider rewriting the whole documentation to make it clear, with all expected input, expected output, updated examples, and so on.